### DISCUSSION TITLE
Teensy Audio System Design Tool

### DISCUSSION THREAD

**Discussion by helste64** [pschatzmann/arduino-audio-tools#327] [Q&A] [answered]
Any thoughts from your side that the Teensy Audio System Design Tool can be used for your audio kit / audio tools.

https://www.pjrc.com/teensy/gui/index.html

**Comment by pschatzmann** [comment, marked as answer]
Unfortunately not: My design goal was to provide a similar functionality in a simple to use Arduino style API.
